packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -57,15 +57,13 @@ abstract class OverlayRoute<T> extends Route<T> {
   bool didPop(T result) {
     final bool returnValue = super.didPop(result);
     assert(returnValue);
-    if (finishedWhenPopped)
```

Comment:
Seems like dartfmt did a bunch of changes on this file. Can you clean it up a bit to just have your changes?